### ISSUE TITLE
[ SECURITY ] Deployment site shows not secured and redirects

### ISSUE THREAD

**Issue by rohan-kulkarni-25** [EddieHubCommunity/LinkFree#197]
![image](https://user-images.githubusercontent.com/76584243/133448183-98736cdd-f6ad-436d-9931-d52d38457571.png)

**Comment by FayasNoushad** [comment]
Yeah, You are right. Not enforced `https`.  Site not provided a layer of encryption.

**Comment by Cahllagerfeld** [comment]
I think thats an issue with the k8s-cluster. Not sure if this can be solved with a PR.

**Comment by krishguptadev** [comment]
This can't be solved with a pr.. I think we need a ssl certificate for this

**Comment by mikeysan** [comment]
Where are we with this? I got a warning from Google Chrome today asking if I wanted to run or proceed with visiting the site.

**Comment by KendallDoesCoding** [comment]
> Where are we with this? I got a warning from Google Chrome today asking if I wanted to run or proceed with visiting the site.

Well, that mostly happens unless it detects some virus or something on the website.. weird.

**Comment by KendallDoesCoding** [comment]
> > I will reach out to some people who could help with this. I also found an potential example
> > https://www.digitalocean.com/community/questions/is-there-a-managed-load-balancer-with-https-support-for-digitalocean-kubernetes
> 
> Try doing this, the issue is that you made the site so long back that it only has http and not https://
> 
> https://www.brafton.com/blog/distribution/how-to-convert-http-to-https-a-quick-guide/

What's weird though is that https://www.eddiehub.org/ is secure and linkfree isn't?

**Comment by KendallDoesCoding** [comment]
> > I will reach out to some people who could help with this. I also found an potential example
> > https://www.digitalocean.com/community/questions/is-there-a-managed-load-balancer-with-https-support-for-digitalocean-kubernetes
> 
> Try doing this, the issue is that you made the site so long back that it only has http and not https://
> 
> https://www.brafton.com/blog/distribution/how-to-convert-http-to-https-a-quick-guide/

Or use a free SSL as suggested by @[praveenscience](https://github.com/praveenscience)

**Comment by eddiejaoude** [comment]
Thank you for raising and the collaboration 👍 

We now have SSL for our EddieHub projects on our Kubernetes cluster
